### PR TITLE
add multiple profiles

### LINKED ISSUE #564: timburgan
1-timburgan

### LINKED ISSUE #568: afc163

### LINKED ISSUE #569: leereilly

### LINKED ISSUE #570: ThalesBMC

### LINKED ISSUE #571: shpatrickguo

### LINKED ISSUE #572: nellyXinwei

### LINKED ISSUE #573: Matteas-Eden

### LINKED ISSUE #574: L0um15

### LINKED ISSUE #575: Israel-Laguan

### LINKED ISSUE #576: bdougie

### LINKED ISSUE #577: evaristocuesta

### PR DESCRIPTION
**Pull request by ozansulukpinar**
resolve #564
resolve #568
resolve #569
resolve #570
resolve #571
resolve #572
resolve #573
resolve #574
resolve #575
resolve #576
resolve #577

### CODE REVIEW

**Comment by schmelto** [comment]
I have to take a closer look when I'm home but I think it would be better if there would be an issue for each of those profiles. But this is only my opinion may we can discuss this and someone else can share their thoughts on this.

**Comment by ozansulukpinar** [comment]
> I have to take a closer look when I'm home but I think it would be better if there would be an issue for each of those profiles. But this is only my opinion may we can discuss this and someone else can share their thoughts on this.

I revised my issue and pull request like what you suggest. You can check again

File: data.json [hunk 1]
```diff
@@ -1018,5 +1018,71 @@
...
+    "name": "Tim Burgan",
+    "githubUsername": "timburgan",
+    "imageUrl": "https://user-images.githubusercontent.com/52232302/124358657-f0673e80-dc29-11eb-9e43-84c889b175b0.jpg",
+    "issueNumber": 564
+  },
+  {
+    "name": "afc163",
+    "githubUsername": "afc163",
+    "imageUrl": "https://user-images.githubusercontent.com/52232302/124358687-1987cf00-dc2a-11eb-9e6f-faa4f425476f.jpg",
+    "issueNumber": 564
```
Comment (schmelto) [inline data.json:1032]: Issue numbers has to be updated here as well otherwise its looking good for me

**Comment by ozansulukpinar** [comment]
It is done.

**Review by schmelto — APPROVED** [review]

**Pull request merged**